test/core/room.test.ts

```diff
@@ -514,4 +515,11 @@ describe('Room', () => {
     expect(attached).toBe(false);
     expect(attachedError).toBe(true);
   });
+
+  it<TestContext>('can be released immediately without unhandled rejections', async (context) => {
+    const room = context.getRoom(defaultRoomOptions);
+
+    // Release the room
+    await (room as DefaultRoom).release();
```

Comment:
same as before - does this test actually fail if there are unhandled rejections?

Reply:
As above